contract-tests/tests/test/amazon/botocore/botocore_test.py

```diff
@@ -372,6 +377,132 @@ def test_kinesis_fault(self):
             span_name="Kinesis.PutRecord",
         )
 
+    def test_bedrock_runtime_invoke_model(self):
+        self.do_test_requests(
+            "bedrock/invokemodel/invoke-model",
+            "GET",
+            200,
+            0,
+            0,
+            rpc_service="Bedrock Runtime",
```

Comment:
Q: why this is required for Bedrock/runtime service but not for the other existing supported AWS resources? such as Kinesis, etc.

Reply:
As in PR description:

> **`_assert_semantic_conventions_span_attributes` function change:**
> In [_assert_semantic_conventions_span_attributes function](https://github.com/aws-observability/aws-otel-python-instrumentation/blob/1753bbf2c3cd41778abc358a7f1e3199e48c7fa9/contract-tests/tests/test/amazon/botocore/botocore_test.py#L448) it is checking the input `service` equals to `rpc.service`, however, we pass the input service with ["remote_service"](https://github.com/aws-observability/aws-otel-python-instrumentation/blob/1753bbf2c3cd41778abc358a7f1e3199e48c7fa9/contract-tests/tests/test/amazon/botocore/botocore_test.py#L430), where there is mismatch for example for Bedrock Agent Runtime: we have
> `rpc_service="Bedrock Agent Runtime", remote_service="AWS::Bedrock". `
> Thus we change to use `rpc_service` if it is provided by:
> ```
> kwargs.get("rpc_service") if "rpc_service" in kwargs else kwargs.get("remote_service").split("::")[-1],
> ```

Whereas, for the other services, for example Kinesis, the `remote_service` is the same as `rpc_service`

Reply:
The reason it is not needed for S3 and other supported services because the `rpc_service` value always matches the second part(after `::`) of value in `remote_service`?

Reply:
Yes.